scripts/utilities.js

```diff
@@ -83,10 +85,39 @@ function shouldContinuePrompt(callback = null) {
   }
 }
 
+// Udpate the meeting demo to the most up to date version of the SDK
+const updateJSSdk = (args) => {
```

Comment:
Wonder if we need these 2 functions? Feel like we could just have one updateDependency where the parameter can be to either install from NPM or from a tar file from a path.